### DISCUSSION TITLE
Added ssd disk write and still maintain the original basis

### DISCUSSION THREAD

**Discussion by b6121061** [Chia-Network/chia-blockchain#6472] [Q&A]
4t nvme 8 cards are concurrently 1000/m. 8tnvme has 16 disk writes that are still 1000/m. What's the problem?
cpu 5950x
128g 
x570
8t nvme  Two 2t nvme are on the motherboard, and four 1t nvme are display cards 4x4
8 photos is the best speed. Adding the overall speed will decrease and increasing the ssd is useless, please tell me why

**Comment by heus12** [comment]
what is X570？

**Reply by b6121061** [reply]
mainboard
